Gruntfile.coffee

```diff
@@ -1299,45 +1303,47 @@ taskManager.add 'socksEchoIntegrationTest', [
   'jasmine_chromeapp:socksEcho'
 ]
 
+taskManager.add 'lint', [
+  'tslint'
+  'jshint'
+]
+
+taskManager.add 'unit_test', [
+  'test_lib'
+  'test_core'
+  'test_ui'
+  'test_chrome'
+]
+
 # TODO: add test_chrome once it passes reliably
 taskManager.add 'integration_test', [
   'tcpIntegrationTest'
   'socksEchoIntegrationTest'
 ]
 
-taskManager.add 'everything', [
-  'build'
-  'test'
-  'integration_test'
-]
-
-# This is the target run by Travis. Targets in here should run locally
-# and on Travis/Sauce Labs.
 taskManager.add 'test', [
-  'exec:lintFirefoxJs'
-  'test_lib'
-  'test_core'
-  'test_ui'
-  'test_chrome'
+  'unit_test'
+  'integration_test'
 ]
 
+# Builds all code, including the "dist" build, but skips
+# linging and testing which can both be annoying and slow.
```

Comment:
I think the only thing stopping things like f4f00114a8737c239866ff45308194bc0f80f163 in the `src/firefox/lib/` source is jshint. Shouldn't a blatant syntax error like that in such an important file as glue.js cause a failure at build time? Otherwise a more obscure failure happens quietly at runtime, and is harder + requires manual testing to notice, if you're not specifically looking for it (especially if you're not running a dev build via `jpm run`).